### ISSUE TITLE
Failure on non-ascii characters of CPL AnnotationText element

### ISSUE THREAD

**Issue by matmat** [Ymagis/ClairMeta#137]
I have an CPL with the following element:

`<AnnotationText>Magnus är sexist (burnt in subtitles)</AnnotationText>`

Clairmeta fails with the following:

```
	check_cpl_contenttitle_annotationtext_match - Check unknown error
Traceback (most recent call last):
  File "clairmeta/dcp_check_base.py", line 118, in run_check
    check_res = func(*args)
  File "clairmeta/dcp_check_cpl.py", line 87, in check_cpl_contenttitle_annotationtext_match
    "mismatch : {} / {}".format(ct, at))
UnicodeEncodeError: 'ascii' codec can't encode character u'\xe4' in position 7: ordinal not in range(128)

	check_cpl_contenttitle_pklannotationtext_match - Check unknown error
Traceback (most recent call last):
  File "clairmeta/dcp_check_base.py", line 118, in run_check
    check_res = func(*args)
  File "clairmeta/dcp_check_cpl.py", line 111, in check_cpl_contenttitle_pklannotationtext_match
    "AnnotationText mismatch : {} / {}".format(ct, at))
UnicodeEncodeError: 'ascii' codec can't encode character u'\xe4' in position 7: ordinal not in range(128)
```

**Comment by remia** [comment]
This is probably due to Python2 not supporting unicode string literals by default nor constructing exception from unicode string.

Quickest fix is to use Python3, I did an update in #138 to support this in Python2 as well, for noz I only activated string literals in the dcp_check_cpl module.

**Comment by remia** [comment]
Fix was merged into develop #138
